benchmark/process/next-tick-exec-args.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 const common = require('../common.js');
 const bench = common.createBenchmark(main, {
-  n: [5e6]
+  n: [4e6]
```

Comment:
These were changed to match those found in the other breadth benchmarks and it also seems to provide more stable results.